network-api/networkapi/wagtailpages/factory/publication.py

```diff
@@ -38,48 +31,49 @@ class CollectionMemberFactory(DjangoModelFactory):
 class DocumentFactory(CollectionMemberFactory):
     class Meta:
         model = get_document_model()
-    title = Faker('text', max_nb_chars=250)
-    file = django.FileField(filename=Faker('file_name', category="text"), file_extension="pdf")
+
+    title = Faker("text", max_nb_chars=250)
+    file = django.FileField(
+        filename=Faker("file_name", category="text"), file_extension="pdf"
+    )
 
 
 class PublicationPageFactory(PageFactory):
-    title = Faker('text', max_nb_chars=120)
-    subtitle = Faker('text', max_nb_chars=250)
-    secondary_subtitle = Faker('text', max_nb_chars=250)
-    publication_date = Faker('date_object')
+    title = Faker("text", max_nb_chars=120)
+    subtitle = Faker("text", max_nb_chars=250)
+    secondary_subtitle = Faker("text", max_nb_chars=250)
+    publication_date = Faker("date_object")
     hero_image = SubFactory(ImageFactory)
     publication_file = DocumentFactory()
 
     class Meta:
         model = PublicationPage
 
 
+# this is actually just a blog author factory, hello tech debt
```

Comment:
if this is tech debt, let's file an issue that explains what's going on here, and what clean up steps would look like, and then point to that issue in this comment instead. It's a fun comment now, but less fun 3 months from now =)

Reply:
Good call. Issue opened #5256